independent-publisher-connectors/OK dokument/SECURITY.md

```diff
@@ -0,0 +1,41 @@
+<!-- BEGIN MICROSOFT SECURITY.MD V0.0.7 BLOCK -->

```

Comment:
@SoliteaSlovensko this file is not required in the open source repository. you can remove it.